franklab_mountainsort/ms4_franklab_proc2py.py

```diff
@@ -310,7 +310,7 @@ def pyms_merge_burst_parents(firings, metrics, firings_out, opts=None):
 def tagged_curation(cluster_metrics, metrics_tagged,
                     firing_rate_thresh=0.01, isolation_thresh=0.95,
                     noise_overlap_thresh=0.03, peak_snr_thresh=1.5,
-                    mv2file='', opts=None):
+                    mv2file='', manual_only=True):
```

Comment:
What does manual only mean?

Reply:
I actually deleted this section as I am not using the "processor". I am calling functions directlt.